### DISCUSSION TITLE
Weather Widget - Wind Direction

### DISCUSSION THREAD

**Discussion by albansuser** [osmandapp/OsmAnd#18110] [Ideas]
The weather plugin and related widgets are a welcome and useful addition to OSMAnd. Thanks for implementing it.

However in addition to windspeed information displayed in the widgets, it would be very useful to also have wind direction displayed. This is especially so for bicycle touring.

Is this feasible?

Cheers

**Comment by abernier** [comment]
indeed, wind direction would be awesome for cyclists ;)
